From 6768f5febc88f4b8c993e090b8405dcc15cd9fb5 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Wed, 12 Aug 2020 10:14:49 +0800
Subject: [PATCH] ARM: dts: rv1126: Add rockchip,temp-freq-table

Add support to change cpu, npu and vepu frequencies according to
temperature.

Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
Change-Id: Ib879164cf719858be7a2f7e1a67bd17d0a6657cc
---
 arch/arm/boot/dts/rv1126.dtsi | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/arm/boot/dts/rv1126.dtsi b/arch/arm/boot/dts/rv1126.dtsi
index 74591090bc5e..9520c9cde04c 100644
--- a/arch/arm/boot/dts/rv1126.dtsi
+++ b/arch/arm/boot/dts/rv1126.dtsi
@@ -111,6 +111,10 @@
 		opp-shared;
 		rockchip,reboot-freq = <816000>;
 
+		rockchip,temp-freq-table = <
+			100000	1296000
+		>;
+
 		opp-408000000 {
 			opp-hz = /bits/ 64 <408000000>;
 			opp-microvolt = <725000 725000 1000000>;
@@ -2118,6 +2122,11 @@
 	rkvenc_opp_table: rkvenc-opp-table {
 		compatible = "operating-points-v2";
 
+		rockchip,temp-freq-table = <
+			80000	500000
+			100000	396000
+		>;
+
 		/* The source clock is CLK_VENC_CORE */
 		opp-297000000 {
 			opp-hz = /bits/ 64 <297000000>;
@@ -2310,6 +2319,12 @@
 			1	37
 		>;
 
+		rockchip,temp-freq-table = <
+			80000	600000
+			90000	396000
+			100000	300000
+		>;
+
 		opp-200000000 {
 			opp-hz = /bits/ 64 <200000000>;
 			opp-microvolt = <725000 725000 1000000>;
-- 
2.35.3

